docs/python-development.rst

```diff
@@ -0,0 +1,53 @@
+
+Python development
+==================
+
+Many OME repositories use
```

Comment:
This is not formatted correctly see https://ome-contributing--8.org.readthedocs.build/en/8/python-development.html

Reply:
Fixed, thx